docs/serving/debugging-application-issues.md

```diff
@@ -133,8 +133,7 @@ conditions:
 If you see this condition, check the following to continue debugging:
 
 - [Check Pod status](#check-pod-status)
-- [Check application logs](#check-application-logs)
-- [Check Istio routing](#check-clusteringressistio-routing)
+- [Check Istio routing](#check-ingressistio-routing)
```

Comment:
Does this link definitely work?
Is it linking from "### Check Ingress/Istio routing", and if so can we change that title to be one or the other instead of with the `/`?